integration/testnet/network.go

```diff
@@ -720,9 +691,9 @@ func BootstrapNetwork(networkConf NetworkConfig, bootstrapDir string) (*flow.Blo
 		EpochTokenPayout:             cadence.UFix64(0),
 		RewardCut:                    cadence.UFix64(0),
 		CurrentEpochCounter:          cadence.UInt64(epochCounter),
-		NumViewsInEpoch:              cadence.UInt64(epochSetup.FinalView - epochSetup.FirstView - 1),
-		NumViewsInStakingAuction:     cadence.UInt64(100),
-		NumViewsInDKGPhase:           cadence.UInt64(100),
+		NumViewsInEpoch:              cadence.UInt64(numViewsInEpoch),
+		NumViewsInStakingAuction:     cadence.UInt64(numViewsInStakingAuction),
+		NumViewsInDKGPhase:           cadence.UInt64(numViewsInDKGPhase),
```

Comment:
double check these values ("final view" vs "length in views" between protocol and smart contract configs)